### ISSUE TITLE
[FEATURE] LiquidBounce Themes - Custom Shaders

### ISSUE THREAD

**Issue by liquidsquid1** [CCBlueX/LiquidBounce#3141]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

The blur shader used by LiquidBounce should be in the theme so that it can be replaced, and there should be a way that you can add your own shaders to overlay over hud elements like a glow.

### Additional context

n/a

**Comment by superblaubeere27** [comment]
Yes, this is planned. At some point when I am motivated enough I will add support for that ;)

**Comment by MukjepScarlet** [comment]
Good idea...

**Comment by MukjepScarlet** [comment]
But to customize it is almost impossible. Pass what to the UBO? To theme background shader we can use general variables.
